.github/workflows/release_mac.yml

```diff
@@ -60,13 +60,20 @@ jobs:
           sed -i '' 's/name = "onnx"/name = "onnx-weekly"/' 'pyproject.toml'
           export ONNX_PREVIEW_BUILD=1
         fi
-        python -m build --wheel
+        python -m build --wheel --sdist
 
-    - uses: actions/upload-artifact@50769540e7f4bd5e21e526ee35c689e35e0d6874
+    - name: Upload wheel artifacts
+      uses: actions/upload-artifact@50769540e7f4bd5e21e526ee35c689e35e0d6874
       with:
         name: wheels-${{ inputs.os }}-${{ matrix.python-version }}-${{ matrix.target-architecture }}
         path: dist/*.whl
 
+    - name: Upload source distribution
+      uses: actions/upload-artifact@50769540e7f4bd5e21e526ee35c689e35e0d6874
+      with:
+        name: sdist-${{ inputs.os }}-${{ matrix.python-version }}-${{ matrix.target-architecture }}
```

Comment:
Do we need any of these in sdist? Python version, os etc.

Reply:
I'll change the name. It's the same for all os. I also had to add the file to download step before publishing.

Reply:
I adapted the name

Reply:
Looks the same but maybe just missing a push?